### PR TITLE
require yaml mime for ManagedReference.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs [hunk 1]
```diff
@@ -131,8 +131,6 @@ public override ProcessingPriority GetProcessingPriority(FileAndType file)
                         {
                             case YamlMime.ManagedReference:
                                 return ProcessingPriority.Normal;
-                            case null:
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Build.ManagedReference/ManagedReferenceDocumentProcessor.cs]: >null [](start = 33, length = 4)

add doc

**Comment by vicancy** [comment]
:shipit:

**Pull request closed without merging**